### ISSUE TITLE
Android crash due to ignore_ssl prop

### ISSUE THREAD

**Issue by sattaman** [samitha9125/react-native-pinch-new#3]
I am getting a crash originating from this line on Android , latest stable react-native (0.61.5)

`if(request.endpoint.startsWith("https") && !request.ignore_ssl) {`

With the following logs

```
2020-02-28 07:44:54.626 29305-29373/com.XXX.XXX E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #2
    Process: com.XXX.XXX, PID: 29305
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$4.done(AsyncTask.java:399)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:919)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference
        at com.localz.pinch.utils.HttpUtil.prepareRequest(HttpUtil.java:85)
        at com.localz.pinch.utils.HttpUtil.sendHttpRequest(HttpUtil.java:139)
        at com.localz.RNPinch$FetchTask.doInBackground(RNPinch.java:126)
        at com.localz.RNPinch$FetchTask.doInBackground(RNPinch.java:79)
        at android.os.AsyncTask$3.call(AsyncTask.java:378)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:919) 
```

**Comment by sattaman** [comment]
@samitha9125 any chance you could bump the version and publish so this fix is released?

**Comment by samitha9125** [comment]
@sattaman I'm so sorry. I totally forgot that. I just published the latest version 0.1.12 with new changes :) Thank you for your contribution.

**Comment by sattaman** [comment]
Thank you :)
